src/tvm_wrapper.cu

```diff
@@ -47,6 +48,26 @@ using namespace flashinfer;
     LOG(FATAL) << "Unsupported data type " << dl_dtype.code; \
   }
 
+#define DISPATCH_REJECTIVE_SAMPLING_NUM_ROUNDS(num_rounds, NUM_ROUNDS, ...)         \
+  if (num_rounds == 1) {                                                            \
```

Comment:
Considering we have early stop, perhaps we don't need each of them?
Something like 1, 2, 4, 8, 16 looks good to me.

Reply:
Just updated